model/ftn/ww3_gint.ftn

```diff
@@ -21,6 +21,7 @@
 !/    31-Aug-2018 : Update groups 2,4,6,8 (S. Zieger)   ( version 6.05 )
 !/    26-Jan-2021 : Added TP field (derived from FP)    ( version 7.12 )
 !/    22-Mar-2021 : New coupling fields output          ( version 7.xx )
+!/    02-Jun-2021 : Big fixs (*SUMGRD; Q. Liu)          ( version 7.xx )
```

Comment:
typo: Bug fix

Reply:
Thanks @aliabdolali. I have corrected this typo.